Active / Inactive Location Icon

lennet opened the pull request:
fixes #44 
![2019-02-16 15-53-56 2019-02-16 15_54_53](https://user-images.githubusercontent.com/7677738/52901406-5f7f9c00-3203-11e9-80fe-08ef79c3b9e2.gif)

lennet commented:
How about having a third state: follow + rotation ?

zutrinken commented:
@lennet there is a dedicated arrow icon for following. Instead of having the line above the tip I put it inside. You just have to rotate the icon so it's pointing north.

![bildschirmfoto 2019-02-20 um 11 29 25](https://user-images.githubusercontent.com/888679/53085391-0feee800-3503-11e9-94e9-c9f44079b504.png)

lennet commented:
@zutrinken I added the third mode and implemented a small scale transition. What do you think?
![2019-03-04 19-19-51 2019-03-04 19_20_57](https://user-images.githubusercontent.com/7677738/53753928-c3b48800-3eb2-11e9-8ad6-fc7756af73ce.gif)
